client/src/reducers/index.js

```diff
@@ -1,21 +1,33 @@
-import { 
-  SET_USER, 
-} from "../constants/action-types";
+import { AUTH_LOGIN, AUTH_LOGOUT, SET_USER } from "../constants/action-types";
 
 const initialState = {
```

Comment:
Should `initialState` also store session errors?

Reply:
Don't think so, could be useful but currently didn't see much error handling, I think it should happen mostly in the forms and perhaps show a toaster in some cases when errors happen from network requests.